services/__tests__/tilesets.test.js

```diff
@@ -10,20 +10,37 @@ beforeEach(() => {
 
 describe('listTilesets', () => {
   test('works', () => {
-    tilesets.listTilesets();
+    tilesets.listTilesets({ ownerId: 'specialguy' });
```

Comment:
originally we had two unit tests, one without ownerId one with, any reason we can't keep both tests?

Reply:
Going by @danswick's comments, I thought it might not be valid to have it any more but I'm not sure. 

So if we have a test without `ownerId` that just shows we have an empty params and query object in the result, is that what we'd want from a `works` test? Or should `works` be the one with an `ownerId`? 

Should I update the validation in listTilesets to use `ownerId: v.required(v.string)`?

Reply:
ownerId should be optional here because although the API requires it https://github.com/mapbox/mapbox-sdk-js/blob/master/lib/classes/mapi-request.js#L127 set's it based on the token if not supplied.

We should have two unit tests here to make sure both omitting and supplying it works.

Reply:
Test reinstated. There has been a change from an `undefined` value when there are no params to an empty object, which would be trivial to change in the `tilesets` service. 

Apologies for there being so many things to change in such a small PR!

Reply:
Ack! Good point, @andrewharvey. And I apologize for leading you astray, @chooban. Poking around a bit more, I don't believe this endpoint supports listing tilesets for accounts other than the one associated with the token provided. I'm double checking with the team who works on that API to be extra sure, but am consistently getting 404 responses from the `list tilesets` endpoint, even for accounts I know have public tilesets. So we may still want to omit it as a param, but not for the reason I originally gave you.

Reply:
No problem at all! Hopefully I can take what I've learned in this PR and add a few more parts of the API.

So at the moment we've got a unit test with no ownerId, and one with. Do we pause until there's an answer from the API team?

Reply:
Let's see what @danswick says but I think we should go ahead with this PR leaving ownerId unaffected, that is unchanged from the current behaviour, with the intention to keep the PR small and focused on the original intent. and if it turns out we shouldn't expose ownerId as an option at all, then we can follow up with a separate PR.

Reply:
Great suggestion, @andrewharvey. I opened an [issue](https://github.com/mapbox/mapbox-sdk-js/issues/355) to track this at a later time. Let's go ahead here!